### PR TITLE
Fix multiple sensor data load bug .

### PR DESCRIPTION
**Pull request by hardik124**
Modified adapter declaration for better performance.
Shifted Sensor data loading code in a method to avoid code replication,

Fixes issue #[Add issue number here. Note: This will automatically close the issue. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]

#457

Apk [here](https://ufile.io/jrl96)

### CODE REVIEW

File: app/src/main/java/org/fossasia/pslab/sensorfragment/SensorFragmentMain.java [hunk 1]
```diff
@@ -55,37 +56,19 @@ public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         scienceLab = ScienceLabCommon.scienceLab;
```
Comment (viveksb007) [inline app/src/main/java/org/fossasia/pslab/sensorfragment/SensorFragmentMain.java]: Remove extra line

File: app/src/main/java/org/fossasia/pslab/sensorfragment/SensorFragmentMain.java [hunk 2]
```diff
@@ -101,30 +84,8 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
...
-                        for (String s : dataAddress) {
-                            tvData += s + ":" + sensorAddr.get(Integer.parseInt(s)) + "\n";
-                        }
-
-                    }
-                    tvSensorScan.setText(tvData);
-                    String[] dataDisp = dataName.toArray(new String[dataName.size()]);
-                    adapter = new ArrayAdapter<>(getContext(), R.layout.sensor_list_item, R.id.tv_sensor_list_item, dataDisp);
-                    lvSensor.setAdapter(adapter);
+                if (!scienceLab.isConnected()) {
```
Comment (viveksb007) [inline app/src/main/java/org/fossasia/pslab/sensorfragment/SensorFragmentMain.java]: ```sciencelab.isConnected()``` returns ```true``` when device is connected. That's when sensor list should be populated.

Comment (hardik124) [inline app/src/main/java/org/fossasia/pslab/sensorfragment/SensorFragmentMain.java]: The data list is populated only after isConnected , here it is only being initialized.

Comment (hardik124) [inline app/src/main/java/org/fossasia/pslab/sensorfragment/SensorFragmentMain.java]: I think , i need to make another PR , because , the updation done by merging the master has removed my code.

Comment (hardik124) [inline app/src/main/java/org/fossasia/pslab/sensorfragment/SensorFragmentMain.java]: Yeah you are right , I did it for checking . and forgot to revert back
